source/command/msxdos/Makefile

```
@@ -20,22 +14,20 @@ endef
 
 define assemble
 	@printf "\n\033[0;36mAssembling %s\033[0m\n\n" $(1)
-	@$(N80) $(1) $(2)
+	@$(N80) $(N80_ARGS) $(1) $(2) $(3)
 endef
 
-nextor-sys: NEXTOR.SYS NEXTORK.SYS
+nextor-sys: prerequisites NEXTOR.SYS NEXTORK.SYS
```

checkmake (0.3.2)

[warning] Target "nextor-sys" should be declared PHONY.

(phonydeclared)